### PR TITLE
Wallendocker

### PR DESCRIPTION
**Pull request by Guaris**

### CODE REVIEW

File: docs/applications/containers/how-to-install-docker-images.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+modified: Wednesday, June 28, 2017
+modified_by:
+  name: Linode
+published: 'Wednesday, June 28, 2017'
+title: 'How to Install Docker and Pull Images '
+external_resources:
+ - '[Docker Docs](http://docs.docker.com/)'
+ - '[Docker Try it Tutorial](https://www.docker.com/tryit/)'
+ - '[Docker Hub](https://hub.docker.com/)'
+
```
Comment (alexfornuto) [inline docs/applications/containers/how-to-install-docker-images.md]: Why the empty space?

bitmoji

File: docs/applications/containers/how-to-install-docker-images.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+This tutorial will walk you through the installation of Docker, and the first steps of pulling down images with the intention of spinning them up as containers. 
+
+
+### Getting Started
+
+In order to install Docker, you must first gain shell access to your Linux deployment. To do this, you must know your Linode IP address. To find that, log into your Linode account and click on the Linodes tab. Within this window, you'll see the **Remote Access tab**; click on that tab to reveal the public IP of your Linode. 
+
+![Remote_Access](/docs/assets/Docker/dockerisntall_a.jpg)
+
+Use the IP to `ssh` into your Linode:
```
Comment (alexfornuto) [inline docs/applications/containers/how-to-install-docker-images.md]: I dunno if this is still the case, but IIRC, numbered steps are king. No numbers, no-go. 

bitmoji

File: docs/applications/containers/how-to-install-docker-images.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+### Getting Started
+
+In order to install Docker, you must first gain shell access to your Linux deployment. To do this, you must know your Linode IP address. To find that, log into your Linode account and click on the Linodes tab. Within this window, you'll see the **Remote Access tab**; click on that tab to reveal the public IP of your Linode. 
+
+![Remote_Access](/docs/assets/Docker/dockerisntall_a.jpg)
+
+Use the IP to `ssh` into your Linode: 
+
+	ssh root@public_ip
```
Comment (alexfornuto) [inline docs/applications/containers/how-to-install-docker-images.md]: Your guide assumes root access, but doesn't [specify it](https://www.linode.com/docs/linode-writers-formatting-guide#include-a-note-about-root-or-non-root-users). Did you read that guide at all?

bitmoji

**Comment by Guaris** [comment]
@alexfornuto Thanks for offering your opinion on this user-contributed guide. Most of your concerns will be addressed in the technical editing stage of this document. If you have any direct suggestions, I invite you to submit a pull request once the guide is finished. 
The documentation team at Linode accepts contributions from authors from various technical  backgrounds. Sometimes we are handed documents that are written in M$ Word, or Google Docs, and need to convert them to markdown before they can be edited. The conversion process usually produces draft documents similar to this one. In conclusion, your opinions and concerns are valid, but premature.  Interesting pictures.

**Comment by alexfornuto** [comment]
bitmoji

**Pull request merged**